player/lua/console.lua

```diff
@@ -524,8 +535,8 @@ local function update()
 
     local screenx, screeny = get_scaled_osd_dimensions()
 
-    local marginx = mp.get_property_native('osd-margin-x')
-    local marginy = mp.get_property_native('osd-margin-y')
+    local marginx = get_margin_x()
+    local marginy = get_margin_y()
```

Comment:
Unrelated, but a thought for the future. We are not consistent with the naming, `get_margin_y`, `margin_y`, `marginy`. I think in such cases, it could be dict with `margin.y` to make it clear or `margin_y` to be consistent with general naming.